.circleci/config.yml

```diff
@@ -145,6 +145,14 @@ jobs:
                 name: Check that the tutorial-setup patches apply
                 command: |
                     scripts/tutorial-setup.sh
+    documentation-check:
+        executor: main-env
+        steps:
+            - checkout
+            - run:
+                name: Check that documentation builds with no warnings/errors
+                command: |
+                    make -C docs html
```

Comment:
Are you sure this actually gives a failing error code if there are only warnings?

Reply:
Yes because that's how I fixed the other documentation errors (missing references to labels).